tortoise/organizer/organizer.go

```diff
@@ -0,0 +1,122 @@
+package organizer
+
+import (
+	"context"
+
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+// LayersIterator is used to traverse organized layers.
+// For siplicity LayersIterator iterates over all layers unconditionally.
+type LayersIterator func(types.LayerID)
```

Comment:
the naming suggests that it's an iterator but the code looks like Organize is the actual iterator object.
it seems this is meant to be the function to execute when layer can found in consecutive order.

how about `LayerCallback` or something...